model/host/static_hosts.go

```diff
@@ -14,17 +14,24 @@ import (
 //
 // Previously this oepration marked these hosts as "decommissioned,"
 // which is not a state that makes sense for static hosts.
-func MarkInactiveStaticHosts(activeStaticHosts []string) error {
+func MarkInactiveStaticHosts(activeStaticHosts []string, distro string) error {
 	if activeStaticHosts == nil {
 		return nil
 	}
-	err := UpdateAll(
-		bson.M{
-			IdKey: bson.M{
-				"$nin": activeStaticHosts,
-			},
-			ProviderKey: evergreen.HostTypeStatic,
+
+	query := bson.M{
+		IdKey: bson.M{
+			"$nin": activeStaticHosts,
```

Comment:
How could these hosts be both not in the list of static hosts and have the type `evergreen.HostTypeStatic`?